src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Hl7v2/ADT_A01/ADT01-28-expected.json

```diff
@@ -23,23 +27,37 @@
         "url": "Organization/2d7d77cd-0ca7-13a5-c9ed-6e70b739adb4"
       }
     },
+    {
+      "fullUrl": "urn:uuid:",
+      "resource": {
+        "resourceType": "Organization",
+        "identifier": [
+          {
+            "value": "GOOD HEALTH HOSPITAL"
+          }
+        ],
+        "destination": [
+          {
+            "endpoint": "GOOD HEALTH HOSPITAL"
+          }
+        ]
+      },
+      "request": {
+        "method": "PUT",
+        "url": "Organization/"
```

Comment:
Why is there no resource id?

Reply:
If we can't generate ID for some resources, we just erase the ID property while keeping other information.